conv_rnn/model.py

```diff
@@ -34,8 +34,6 @@ def __init__(self, word_model, **config):
         else:
             raise ValueError("RNN type must be one of LSTM or GRU")
         self.conv = nn.Conv2d(1, n_fmaps, (1, self.hidden_size * 2))
-        if dropout:

```

Comment:
does this hurt performance? if we are removing it why are we adding `--dropout_prob` to `train.py`?

Reply:
Paper says no dropout for SST1/2.